src/CollectionMacros.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+
+namespace Calebporzio\AwesomeHelpers;
+
+
+class CollectionMacros
+{
+    public function toAssoc()
+    {
+        return function (){
```

Comment:
I think we should add the white space before the `(` character because of consistency.